src/main/resources/view/StatsCard.fxml

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can we verify this additional line?

Reply:
This will be a future addition for some stats in the top half of the GUI.